Source/WebCore/DerivedSources.make

```diff
@@ -384,7 +384,6 @@ JS_BINDING_IDLS := \
     $(WebCore)/Modules/identity/DigitalCredentialRequestOptions.idl \
     $(WebCore)/Modules/identity/OpenID4VPRequest.idl \
     $(WebCore)/Modules/identity/IdentityCredentialProtocol.idl \
-    $(WebCore)/Modules/identity/Navigator+Identity.idl \
```

Comment:
You'll also have to remove the file reference from `Source/WebCore/CMakeLists.txt` to unblock the GTK build.